test/Dashboard.UnitTests/RestApiTests.cs

```diff
@@ -36,21 +36,33 @@ public RestApiTests(Fixture fixture)
         [Fact]
         public async Task GetDefinition()
         {
-            
-                var response = await _client.GetJsonAsync<FunctionStatisticsSegment>(_endpoint + "/api/functions/definitions?limit=100");
-
-                // Testing against specific data that we added. 
-                Assert.Equal(null, response.ContinuationToken);
-                var x = response.Entries.ToArray();
-                Assert.Equal(1, x.Length);
-                Assert.Equal("alpha", x[0].functionName);
+            var response = await _client.GetJsonAsync<FunctionStatisticsSegment>(_endpoint + "/api/functions/definitions?limit=100");
+
+            // Testing against specific data that we added. 
+            Assert.Equal(null, response.ContinuationToken);
+            var x = response.Entries.ToArray();
+            Assert.Equal(1, x.Length);
+            Assert.Equal("alpha", x[0].functionName);
+        }
+
+        // Pass ?host=  parameter when getting definitions to filter to a specific host. 
+        [Fact]
+        public async Task GetDefinitionSingleHost()
+        {
+            var response = await _client.GetJsonAsync<FunctionStatisticsSegment>(_endpoint + "/api/functions/definitions?limit=100&host=missing");
```

Comment:
host id is case insensitive.

---
In reply to: [84857185](https://github.com/Azure/azure-webjobs-sdk/pull/884#discussion_r84857185) [](ancestors = 84857185)